.github/workflows/release.yml

```diff
@@ -167,3 +158,18 @@ jobs:
           download-url: https://github.com/railwayapp/cli/releases/latest/download/railway-${{ needs.create-release.outputs.railway_version }}-${{ matrix.target }}.tar.gz
         env:
           COMMITTER_TOKEN: ${{ secrets.COMMITTER_TOKEN }}
+  publish-crate:
```

Comment:
I don't think this is needed. We publish with `cargo release` which automatically publishes to crates.io

Reply:
Unless the intention is to change the release process and initiate it via GitHub?

Reply:
yes it is, having it run locally imo is a bad idea

Reply:
Yup agreed it isn't ideal to run locally. So the flow will be
- Manually push a new Git tag
- Release action is triggered which creates a new GH release based on tag version and builds binaries for all platforms
- Publish action is triggered on release which publishes a new version to crates.io
?

Once this flow is finalized could you please document it in the [contributing.md](https://github.com/railwayapp/cli/blob/master/CONTRIBUTING.md) file.

Reply:
One benefit of `cargo release` is that it would automatically bump the version in `Cargo.toml`. Will that be a manual process now?

Reply:
Nayural process now, and the tag is automaticalmy crated.

Reply:
I might be missing something, but what automatically creates the tag?

Reply:
[- name: Create GitHub release](https://github.com/railwayapp/cli/blob/f0acca3c0c8db89e49baf8820037f3fc31c4fb15/.github/workflows/release.yml#L40-L47)

Reply:
That workflow runs when there is a new tag, https://github.com/railwayapp/cli/blob/f0acca3c0c8db89e49baf8820037f3fc31c4fb15/.github/workflows/release.yml#L5. What creates the new tag?

Reply:
`git tag vX.X.X`
`git push origin vX.X.X`

Reply:
yeah.. this is starting to get a bit annoying. I'm going to check how I do it in my CI.